libs/platform/experience/src/routes.ts

```diff
@@ -7,6 +7,11 @@ export const defaultExperienceRoutes: RouteConfig[] = [
   {
     pattern: new URLPattern({ pathname: '/{index.html}?' }),
   },
+  {
```

Comment:
Good idea, but I would prefer to keep both implementation (for the route directly, and for the page on EB side) EB will override or get the default value from route. But I can imagine that this is a lot of work, lets skip it for now, because it is a candidate for the big feature